### DISCUSSION TITLE
custom magical tradition can not be saved correctly

### DISCUSSION THREAD

**Discussion by o8livion** [chummer5a/chummer5a#4850] [Q&A] [answered]
### hello!

i appreciate your work and want the project to even get better.
one thing i recognized was with the mages that have a custom tradition. the spirit for healing is not saved, the drainresisting attributes sometimes neither.

am i doing something wrong? does there need to be some specific settings activated?

**EDIT: just figured out, it also applies to preconfigured traditions, tested shamanic and after save and load, health was blanked out too.**

thx n hand

![custom tradition before 1](https://user-images.githubusercontent.com/14218286/178118000-b0154a12-2a69-4c7f-95fd-9699f0ccac00.jpg)
![custom tradition after 1](https://user-images.githubusercontent.com/14218286/178118001-86100e91-0c53-404e-a257-7c3bc3eb2c6e.jpg)
![custom tradition before 2](https://user-images.githubusercontent.com/14218286/178118002-9d16a82a-b0f7-4ce4-8fb9-41b31b133b24.jpg)
![custom tradition after 2](https://user-images.githubusercontent.com/14218286/178118003-667d31e2-e76c-457e-863c-8f4260c17d45.jpg)

**Comment by chummer5a** [comment, marked as answer]
This was resolved in nightlies. Incorrect binding.
